lib/xdr/transactions/operations/bump_sequence.ex

```diff
@@ -1,4 +1,4 @@
-defmodule StellarBase.XDR.Operations.BumpSequence do
+defmodule StellarBase.XDR.BumpSequence do
```

Comment:
For the modules that are operations, don't remove `Operations` from the namespace